crypto-ffi/bindings/js/src/CoreCrypto.ts

```diff
@@ -1150,8 +1128,28 @@ export class CoreCrypto {
         try {
             await this.#cc.provide_transport(
                 new MlsTransportProvider(
-                    transportProvider.sendCommitBundle,
-                    transportProvider.sendMessage,
+                    async (commitBundle: CommitBundleFfi) => {
+                        const result = await transportProvider.sendCommitBundle(
+                            {
+                                welcome: commitBundle.welcome,
+                                commit: commitBundle.commit,
+                                groupInfo: {
+                                    encryptionType:
+                                        commitBundle.group_info.encryption_type,
+                                    ratchetTreeType:
+                                        commitBundle.group_info
+                                            .ratchet_tree_type,
+                                    payload: commitBundle.group_info.payload,
+                                },
+                            }
```

Comment:
Why can't we pass the commitBundle directly? Mismatching type? Shouldn't we also have ffi mapping function here as well then?